index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <span class="my-logo"><a href="#">My Logo</a></span>
+        <nav>
+          <a href=""><img src="assets/images/Union.png" alt="Union-icon" /></a>
+        </nav>
+      </header>
+
+      <section class="hero">
```

Comment:
- [ ] Kindly consider displaying all the contents of this section with Flexbox because that will give you a lot of options to align all your elements, plus, it's responsive.